web-ui/src/main/resources/catalog/components/edit/onlinesrc/OnlineSrcDirective.js

```diff
@@ -1587,8 +1587,12 @@
 
                   // Append * for like search
                   scope.updateParams = function() {
-                    scope.searchObj.params.any =
-                        '*' + scope.searchObj.any + '*';
+                    var addWildcard = scope.searchObj.any.indexOf('"') === -1
+                      && scope.searchObj.any.indexOf('*') === -1
+                      && scope.searchObj.any.indexOf('q(') !== 0;
```

Comment:
Will searching for  
```
q(+resourceTitleObject.default:("corine land cover 2012" AND "version 18"))
```
vs
```
             q(+resourceTitleObject.default:("corine land cover 2012" AND "version 18"))
```
produce different result?

I occasionally accidently add a space before my search criteria.  I would expect that it would not change the results.

Maybe ensure that searchObj is trimmed before the check?

Reply:
Indeed but while testing at least in the associated panel it looks like AngularJS is trimming the model value already (see https://docs.angularjs.org/api/ng/input/input%5Btext%5D) "If set to false AngularJS will not automatically trim the input. This parameter is ignored for input[type=password] controls, which will never trim the input. (default: true)"

So it looks like the input value is already trimmed 

![image](https://user-images.githubusercontent.com/1701393/124755097-3c1c2f80-df2b-11eb-8235-716cf732b704.png)


So sounds ok. Maybe you noticed some others places where the any param is not coming from an input model?

Reply:
Let me know if you have the trim issue somewhere else in the UI and can have a look in another PR. Merging this one.